CTFd/__init__.py

```diff
@@ -271,10 +271,8 @@ def set_sqlite_pragma(dbapi_connection, connection_record):
 
         app.register_blueprint(admin)
 
-        app.register_error_handler(404, page_not_found)
-        app.register_error_handler(403, forbidden)
-        app.register_error_handler(500, general_error)
-        app.register_error_handler(502, gateway_error)
+        for code in {403, 404, 500, 502}:
+            app.register_error_handler(404, render_error)
```

Comment:
I think this should be code right?

Reply:
Yes, of course. ^c^v fail on my part! Thanks for correcting it